crates/autopilot/src/run_loop.rs

```diff
@@ -522,7 +528,7 @@ impl RunLoop {
 
     /// Runs the solver competition, making all configured drivers participate.
     /// Returns all fair solutions sorted by their score (best to worst).
-    async fn competition(&self, auction: &domain::Auction) -> Vec<Participant> {
+    async fn competition(&self, auction: &domain::Auction) -> Vec<competition::Participant> {
```

Comment:
We did not envision implementing unit tests for the runloop functionality (although I agree it could be useful for these small logically interesting parts).

For now I am happy if the changes do not break the current setup (running with 1 winner) which I regularly check on staging. 

Once the thole feature is finished, I think having complete e2e test would be useful before activating the feature gradually on each network.